### DISCUSSION TITLE
Reducing resources using exporter, unable to find inspiration from demo

### DISCUSSION THREAD

**Discussion by freshgeek** [open-telemetry/opentelemetry-java-instrumentation#12154] [Q&A] [answered]
Due to the excessive amount of information reported by the agent and the rising cost for cloud providers, I want to reduce costs by removing some invalid attributes and information, such as resources. However, according to the official documentation, the deletion of resources is done using an exporter, and I cannot find inspiration from the demo
https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/examples/extension/src/main/java/com/example/javaagent/DemoSpanExporter.java

io.opentelemetry.semconv.resource.attributes.ResourceAttributes#PROCESS_COMMAND_LINE
process.command_line

**Comment by xiepuhuan** [comment, marked as answer]
You can use configuration item `-Dotel.experimental.resource.disabled-keys=process.command_line` to disable the resource attribute `process.command_line`

**Reply by xiepuhuan** [reply]
Related documents in https://opentelemetry.io/docs/languages/java/configuration/#resources

**Reply by freshgeek** [reply]
Very perfectly solved my question, thank you very much. I have been looking at the zero configuration document, so I overlooked this document

**Reply by trask** [reply]
> I have been looking at the zero configuration document, so I overlooked this document

cc @jack-berg @svrnm I think the proposed doc restructure will improve this?

**Reply by svrnm** [reply]
@trask I would think so, yes!:-)
